server/core/src/initServer.cpp

```diff
@@ -179,6 +181,8 @@ int initServerInfo(int processType, rsComm_t* rsComm)
         }
     }
 
+    // TODO This function invokes acAclPolicy which guarantees GenQuery1 honors
+    // the permission model. acAclPolicy is invoked the first time _rsGenQuery is invoked.
     status = initZone( rsComm );
```

Comment:
Should probably document this? Then again, we're talking about removing access to the strict ACLs option sooo, I'll just reference `initZone()` in that issue and remove this comment.

Reply:
Left comment about this in #7954.